public/js/action-list.js

```diff
@@ -82,14 +82,31 @@
                 $list.find('.list-item.active').removeClass('active');
                 $item.addClass('active');
             }
+
+            // For items that do not have a bottom status bar like Downtimes, Comments...
+            if (! $list.closest('.container').children('.footer').length) {
+                $list.closest('.container').append(
+                    '<div class="footer" data-action-list-automatically-added></div>'
+                );
+            }
         } else {
             $list.find('.list-item.active').removeClass('active');
             $item.addClass('active');
         }
 
         $activeItems = $list.find('.list-item.active');
+        var container = $list.closest('.container');
+        var bottomStatusBar = container.find('.footer');
```

Comment:
Why not move this further up and use it already in the check on line 87?

Reply:
And remember to only use `find` if you don't know or care about an elements position in the tree. The footer however should only be a direct child of the container, so `children` should suffice.

Reply:
Oh, aaand, you should rename the variable. It's the footer after all, not the status bar.